pro/src/components/CollectiveOfferNavigation/__specs__/CollectiveOfferNavigation.spec.tsx

```diff
@@ -459,13 +465,55 @@ describe('CollectiveOfferNavigation', () => {
     expect(archiveButton).not.to.toBeInTheDocument()
   })
 
+  it('should not see archive button when offer is not archivable when FF status is enable and action is not possible', () => {
+    renderCollectiveOfferNavigation(
+      {
+        ...props,
+        isTemplate: true,
+        isCreatingOffer: false,
+        offer: {
+          ...offer,
+          allowedActions: [],
+        },
+      },
+      { features: ['ENABLE_COLLECTIVE_NEW_STATUSES'] }
+    )
+
+    const archiveButton = screen.queryByRole('button', {
+      name: 'Archiver',
+    })
+
+    expect(archiveButton).not.to.toBeInTheDocument()
+  })
+
+  it('should not see archive button when offer is not archivable when FF status is enable and action is possible', () => {
+    renderCollectiveOfferNavigation(
+      {
+        ...props,
+        isTemplate: true,
+        isCreatingOffer: false,
+        offer: {
+          ...offer,
+          allowedActions: [CollectiveOfferTemplateAllowedAction.CAN_ARCHIVE],
+        },
+      },
+      { features: ['ENABLE_COLLECTIVE_NEW_STATUSES'] }
+    )
+
+    const archiveButton = screen.queryByRole('button', {
+      name: 'Archiver',
+    })
+
+    expect(archiveButton).to.toBeInTheDocument()
```

Comment:
Du coup si on est sur que le bouton est là donc on peut utiliser `getByRole` et aussi il y a un `.to` en trop ligne 507